knimin/templates/pm_targeted_concentration_check.html

```diff
@@ -0,0 +1,185 @@
+      var idx = $("#general-div").data('pm-idx') - 1;
+      loadGUI(plates, idx);
+    });
+
+    $('#pm-next').click(function(e) {
+      var idx = $("#general-div").data('pm-idx') + 1;
+      loadGUI(plates, idx);
+    });
+
+    $('#pm-pool').click(function(e) {
+      var form = $('<form>', {action: '', method: 'post'});
+      $('<input>').attr({type: 'hidden', name: 'plates', value: JSON.stringify(plates)}).appendTo(form);
+      form.appendTo('body').submit();
+    });
+
+    $('#pm-filter-min').change(function () {
+      updateMod(plates[$("#general-div").data('pm-idx')]);
+    });
+
+    $('#pm-filter-max').change(function () {
+      updateMod(plates[$("#general-div").data('pm-idx')]);
+    });
+
+    $('#pm-filter-blanks').change(function () {
+      updateMod(plates[$("#general-div").data('pm-idx')]);
+    });
+
+    $('#pm-norm').change(function () {
+      updateMod(plates[$("#general-div").data('pm-idx')]);
+    });
+
+    $('#pm-epmotion').click(function () {
+      var $form = $('<form>', {action: '/pm_targeted_pool_epmotion/', method: 'post', target:'_blank' });
+      $('<input>').attr({type: 'hidden', 'name': 'plate', value: JSON.stringify(plates[$("#general-div").data('pm-idx')])}).appendTo($form);
+      $('<input>').attr({type: 'hidden', 'name': 'destination', value: $('#pm-destination').val()}).appendTo($form);
+      console.log($('#pm-destination').val());
+      $form.appendTo('body').submit()
+    });
+  });
+</script>
+{% end %}
+{% block content %}
+<div class="container" id="general-div">
+  <div class="row">
+    <div class="col-xs-9 form-group">
+      <label><h3>Check concentration values - Plate: <b id="pm-name"></b> <i id='pm-counter'></i></h3></label>
+    </div>
+    <div class="col-xs-1">
+      <button id='pm-prev' class='btn btn-info'><i class='glyphicon glyphicon-triangle-left'></i> Prev</button>
+    </div>
+    <div class="col-xs-1">
+      <button id='pm-next' class='btn btn-info'>Next <i class='glyphicon glyphicon-triangle-right'></i></button>
+    </div>
+    <div class="col-xs-1">
+      <button id='pm-pool' class='btn btn-success'>Pool <i class='glyphicon glyphicon-fast-forward'></i></button>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-xs-4 form-group">
+      <label>Normalization value (ng):</label>
+      <input type="number" id="pm-norm" min=0 value=240 step=0.001 value=240/>
+    </div>
+    <div class="col-xs-4 form-group">
+      <label>Blanks value (&mu;L):</br>(blank wells will be set to this value)</label>
+      <input type="number" id="pm-filter-blanks" min=0 value=2 step=0.001 />
+    </div>
+    <div class="col-xs-4 form-group">
+      <label>EPMotion destination:</label>
+      <input type="number" id="pm-destination" min=1 max=24 value=1 />
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-xs-4 form-group">
+      <label>Minimum concentration: (ng/&mu;L)</br>(values lower than this will be set to this value)</label>
```

Comment:
This sentence is a little confusing for the user.  What are the values set to when they are below what value?